tide_api.info.yml

```diff
@@ -4,8 +4,8 @@ type: module
 package: Tide
 core: 8.x
 dependencies:
-  - drupal:jsonapi (>=8.x-2.0)
-  - drupal:jsonapi_extras (>=8.x-3.3)
+  - drupal:jsonapi (>=8.x-2.4)
```

Comment:
I didn't actually know you could do this. What actual effect does it have? Will the module fail to enable if the version in composer.json is not consistent?

Reply:
Drupal won't enable the module if its version does not satisfy this requirement.